exercises/simple-linked-list/Example.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+public class SimpleLinkedList<T>
```

Comment:
Looking at other implementations of this, there are some methods that would be nice to have represented in the tests: `Head`, `Push()`, `Pop()`, & `Peek()`. They do `Size()` but since this is .Net, I could go with `Count`, maybe even try implementing `IList`? Your call.